category_theory/examples/monad.lean

```diff
@@ -0,0 +1,31 @@
+
+import category_theory.category
+
+universes u v
+
+namespace category_theory
+
+def Monad (m) [monad.{u v} m] := Type u
```

Comment:
I should have mentioned this earlier but this is called the "Kleisli category" of `m`. I think `Kleisli m` would be a better lean name. (As is, it sounds like `Monad` should be some kind of category of monads, which also exists.)

Reply:
You're right, thanks! Do you think I should put it in `category_theory.instances.kleisli` or `category_theory.instances.monad`?

Reply:
There shouldn't be a problem with lawlessness here since the monad is a parameter to this category, not part of the object data. You can just tack on another assumption if you need it to be lawful.